content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -280,16 +294,32 @@ bundles and concatenates all specs together, in essence running scripts like
 shown below. This means the code in the support file is executed once before all
 spec files, instead of once before each spec file.
 
-```html
```

Comment:
Is this actually even how Cypress works? I was under the impression that it would only run the `supportFile` and then a single spec file. And then, when you told it to load another spec file, it would clear everything out, run the `supportFile` and then the next spec file.

Reply:
This was updated/removed in another PR so no longer relevant.